test/test_coinbase_pro_active_order_tracker.py

```diff
@@ -17,11 +16,11 @@
 from wings.order_book_tracker import OrderBookTrackerDataSourceType
 from wings.order_book_message import (
     CoinbaseProOrderBookMessage,
-    OrderBookMessageType
```

Comment:
change to one line import, my bad accidentally left this in earlier

Reply:
done